src/ctrls/PlaybookBuildCtrl.js

```diff
@@ -29,6 +29,14 @@ class PlaybookBuildCtrl extends Controller{
             if(args[3] === 'usage') {
                 console.log(`If you run ${chalk.magenta('playbook build')} it will look for all the ${chalk.green('*.playbook.js')} files`);  
             }
+            else
+            {
+                global.playbook = require('../playbook.sdk').playbook;
+                global.step = require('../playbook.sdk').step;
+                PlaybookService.buildPlaybookJsonFromFolderPath(args[3])
```

Comment:
- [ ] todo - add task to backlog to change to *yargs* https://www.npmjs.com/package/yargs